pkg/dia/helpers/NameForSymbol.go

```diff
@@ -359,7 +457,7 @@ func NameForSymbol(symbol string) string {
 	case "SYS":
 		return "Syscoin"
 	case "BTM":
-		return "Bytom"
+		return "Bitmark"
 	case "HAL":
```

Comment:
this one has is actually Bytom https://coinhub.diadata.org/coin-details/BTM

Reply:
Actually is [duplicated](https://coinmarketcap.com/search/?q=BTM).
If you want I can update.